packages/tracing/src/transaction.ts

```diff
@@ -235,19 +235,17 @@ export class Transaction extends SpanClass implements TransactionInterface {
         ? rate.toLocaleString('fullwide', { useGrouping: false, maximumFractionDigits: 16 })
         : undefined;
 
-    // For now we're not sending the transaction name and user_id due to PII concerns
-    // commenting it out for now because we'll probably need it in the future
-
     const scope = hub.getScope();
-    const { /* id: user_id, */ segment: user_segment } = (scope && scope.getUser()) || {};
+    const { segment: user_segment } = (scope && scope.getUser()) || {};
+
+    const source = this.metadata.source;
+    const transaction = source && source !== 'url' && source !== 'unknown' ? this.name : undefined;
```

Comment:
Was debating whether to include `unknown` in this condition. According to the spec Relay treats this value equally as transactions without a source. Which IMO means we shouldn't add the transaction name to the DSC.

Reply:
Yes that's a very good point. Thoughts about removing `'unknown'` @AbhiPrasad?